RecordShow.java

```diff
@@ -0,0 +1,113 @@
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.util.Scanner;
+
+class RecordShow {
+    // start Recoding after inserting programme name and time id
+    int user_id;
+
+    void recordShow(Scanner inp, int user_id) throws Exception {
+        user_id = this.user_id;
+        PreparedStatement pst;
+        int rs;
+        ResultSet resultSet;
+        SQLQueries sql = new SQLQueries();
+
+        String program_routine = "select * from program_routine join channel on program_routine.channel_id=channel.channel_id join  time_slot on program_routine.time_slot_id=time_slot.time_slot_id ";
+        pst = sql.con.prepareStatement(program_routine);
+        resultSet = pst.executeQuery();
+        while (resultSet.next()) {
+            System.out.println(resultSet.getInt("program_id") + "\t"
```

Comment:
Why are you manually adding spaces to the display functions, 
There are functions to pad strings in Misc.java
```
padAllRight(String[] arr, int L)
padAllRight(String[] arr, int L, String color) 
```